roles/ec2-s3-bucket/files/lifecycle.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This repo is public. Can you think of any reason why this would be insecure?

Reply:
I *think*, this just tells viewers that we delete items in buckets regularly. (every 30 days).

An attacker gaining this information couldn't do much with that information, correct?